go.mod

```diff
@@ -40,4 +40,6 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 )
 
+replace github.com/gin-gonic/gin v1.6.3 => github.com/gin-gonic/gin v1.7.0
```

Comment:
Why is this required? Just update the go.mod & go.sum it should be good to go as gin is not a direct dep.

Reply:
It is a dep on nhooyr.io/websocke.  It is possible to fix it in our go.mod. Because it always be tidy by `go mod tidy`. replace is easy to fix it.